src/Route.jsx

```diff
@@ -11,15 +11,24 @@ const ICON_MAPPING = {
   },
 };
 
-export function getStopMarkersLayer(route, getStopInfo) {
+/* changes color of selected stop along a route */
+function selectedStopColor(stop, selectedStops) {
+  const selectedStopsIds
+  = selectedStops.map(currentstop => currentstop.sid);
+  if (selectedStopsIds.includes(stop.sid)) {
```

Comment:
@EddyIonescu using includes (syntax is includes not contains :) Same idea though obviously)